libraries/abstractions/pkcs11/mbedtls/iot_pkcs11_mbedtls.c

```diff
@@ -499,20 +499,12 @@ CK_RV prvMbedTLS_Initialize( void )
 {
     CK_RV xResult = CKR_OK;
 
-    if( xP11Context.xIsInitialized == CK_TRUE )
```

Comment:
This change doesn't cause a regression in the auto-generated tests from Galois?

Reply:
No, this statement was never false, as the only function that calls this private function checks this condition before invoking prvMbedTLS_Initialize